docs/jade-getting-started.md

```diff
@@ -52,6 +52,18 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
+### 3. Create Terra Accounts
```

Comment:
Should this be a subheading of section 2 or a separate section?

Reply:
Good catch! I meant to make that a separate section.